crates/forge/src/new.rs

```diff
@@ -15,8 +15,8 @@ use toml_edit::{value, Array, ArrayOfTables, DocumentMut, Item, Table, Value};
 
 static TEMPLATE: Dir = include_dir!("starknet_forge_template");
 
-const DEFAULT_ASSERT_MACROS: Version = Version::new(0, 1, 0);
-const MINIMAL_SCARB_FOR_CORRESPONDING_ASSERT_MACROS: Version = Version::new(2, 8, 0);
+const DEFAULT_ASSERT_MACROS: Version = Version::new(2, 8, 5);
+const MINIMAL_SCARB_FOR_CORRESPONDING_ASSERT_MACROS: Version = Version::new(2, 8, 5);
```

Comment:
This shouldn't change. I know we officially stop support for versions below 2.8.5, but this will break them entirely.

```suggestion
const MINIMAL_SCARB_FOR_CORRESPONDING_ASSERT_MACROS: Version = Version::new(2, 8, 0);
```